WcaOnRails/app/models/team_member.rb

```diff
@@ -8,6 +8,13 @@ def current_member?
     end_date == nil || end_date > Date.today
   end
 
+  validate :start_date_must_be_earlier_than_end_date
+  def start_date_must_be_earlier_than_end_date
+    if end_date != nil && start_date >= end_date
+      errors.add(:start_date, " must be earlier than end_date.")
```

Comment:
There's an uneeded space in this error string. Also, do we normally put periods at the end of our error strings?